src/p2/interpreter.rs

```diff
@@ -70,9 +70,15 @@ impl InterpreterState {
             }
             self.document_stack[l].done_processing_imports();
 
+            // println!("\nBefore ignoring comments !! ===========================================\n");
+            // self.document_stack[l].show_document();
+
```

Comment:
Don't add these comments, delete them.

Reply:
I was just testing this I will remove these comments and that debug function once all conditions are passed in the final commit.